### PR TITLE
Visual regression test

### LINKED ISSUE #4: Add visual regression tests for typesetting libraries
We may want to add tests to confirm that we don't break typesetting libraries (like `derive`, `code2`, ... etc).

My current idea is leveraging existing tools for front-end development as a first step:
* [jest-image-snapshot](https://github.com/americanexpress/jest-image-snapshot)
* [reg-suit](https://github.com/reg-viz/reg-suit)

### PR DESCRIPTION
**Pull request by zeptometer**
Merging this PR will close #4 

**Steps for running integration test:**
1. Install SATySFi, [Yarn](https://yarnpkg.com/lang/ja/docs/install/#debian-stable), and [GraphicsMagick ](http://www.graphicsmagick.org)
2. Run `yarn install` to install dependencies
3. Run `yarn test` to run tests

You can see the basic idea of the visual regression tests in [this article](https://qiita.com/zptmtr/items/59b485c47edfbca67c60). 
Those tests will also run in GitHub Actions.

**Future work:**
* Add instruction on how to add/maintain visual regression tests
* Use appropriate class file instead of StdJa (cf. #14)

### CODE REVIEW

**Comment by zeptometer** [comment]
CI not working because of my wrong configuration🤔 I'll dig into it later.

**Comment by zeptometer** [comment]
Regression tests are passing in GitHub Actions👏

**Comment by zeptometer** [comment]
Now it's ready for review! I'd like feedback especially on
* Does `__test__/README.md` make enough sense to you?
* The existing test in `t/` can be migrated to Jest snapshot tests in `__test__`. Do we wanna do that?

File: __test__/render.test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  expect(shell.exec('satysfi -v').code).toBe(0);
+})
+
+describe('Derive', () => {
+  test('renders derive', compareRenderOutputWithSnapshot('satysrc/derive/01-derive'));
+  test('renders assume', compareRenderOutputWithSnapshot('satysrc/derive/02-assume'));
+  test('renders by', compareRenderOutputWithSnapshot('satysrc/derive/03-by'));
+  test('renders by and byOp', compareRenderOutputWithSnapshot('satysrc/derive/04-by-and-byop'));
+  test('renders from', compareRenderOutputWithSnapshot('satysrc/derive/05-from'));
+  test('renders dotted line', compareRenderOutputWithSnapshot('satysrc/derive/06-dotted-line'));
```
Comment (nyuichi) [inline __test__/render.test.js]: nitpick: Do you plan to add any other types of tests here in the future? Otherwise I would turn this into a for loop.

Comment (zeptometer) [reply __test__/render.test.js]: Good catch👍

**Pull request merged**